### PR TITLE
trivial: fix buyout total cost

### PR DESCRIPTION
**Pull request by adamgobes**

### CODE REVIEW

File: components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx [hunk 1]
```diff
@@ -206,7 +206,17 @@ export function LoanFormBetterTerms({
           }}
           balance={balance}
           accrued={formattedInterestAccrued}
-          totalPayback={formattedTotalPayback}
+          totalPayback={[
+            truncate(
```
Comment (cnasc) [inline components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx:210]: what does this truncate it to?

File: components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx [hunk 2]
```diff
@@ -19,10 +19,10 @@ import { Explainer } from './Explainer';
 import { ethers } from 'ethers';
 import { daysToSecondsBigNum, secondsBigNumToDays } from 'lib/duration';
 import { yupResolver } from '@hookform/resolvers/yup';
-import { Balance } from '../Balance';
 import { formattedAnnualRate } from 'lib/interest';
 import { LoanTermsDisclosure } from 'components/LoanTermsDisclosure';
 import { useLoanDetails } from 'hooks/useLoanDetails';
+import { truncate } from 'lodash';
```
Comment (cnasc) [inline components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx:25]: nit, this style of import causes all of lodash to get imported, should do `import truncate from 'lodash/truncate'` or similar

File: components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx (hunk 1 above)
Comment (adamgobes) [reply components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx:210]: not quite sure i am following the pattern of other formatted amounts from useLoanDetails

Comment (adamgobes) [reply components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx:210]: looks like it adds ... if string length is longer than certain char count

Comment (cnasc) [reply components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx:210]: `useLoanDetails` defines a function coincidentally called `truncate`:
```ts
function truncate(num: string, maxDigits?: number): string;
function truncate(num: number, maxDigits?: number): string;
function truncate(num: string | number, maxDigits: number = 4): string {
  const number = typeof num === 'string' ? parseFloat(num) : num;
  const delta = number - Math.floor(number);
  // TODO: handle very small numbers that may render as 0.0000
  return number.toFixed(delta > 0 ? maxDigits : 0);
}
```

Comment (cnasc) [reply components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx:210]: at any rate, I think Wilson wanted these to render with as many decimals as the asset supports?

Comment (adamgobes) [reply components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx]: oops thanks for that find, that was careless of me, ya lets remove truncate altogether

**Review by cnasc — APPROVED** [review]

**Pull request merged**